ansible/roles/sonic-common/tasks/main.yml

```diff
@@ -18,6 +24,14 @@
 - include: aptrepo.yml
   tags: repo
 
+# Dbus
+- name: Install dbus
+  become: true
+  apt: pkg=dbus
```

Comment:
I need to test on "clean" image once I got it. But it seems DBus  is widely used by systemd: systemct use DBus to communicate with systemd [https://people.debian.org/~stapelberg/docs/systemd-dependencies.html](url)
Uses socket and D-Bus activation for starting services [https://wiki.debian.org/systemd#systemd_-_system_and_service_manager](url)
It can be that other vendors got it when build image.